packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -142,9 +158,7 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
                 return;
             }
             case SummaryManagerState.Starting: {
-                if (shouldSummarizeState.shouldSummarize) {
```

Comment:
Just curious - why remove this check?

Reply:
So, the state machine will never leave the "Starting" state until it finishes the delay. For that reason, we are giving ourselves a chance to change states sooner, whether or not that is moving into "Running" or stopping, etc.